src/field.cpp

```diff
@@ -1026,7 +1108,66 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         // Below we will access our nearest 8 neighbors, so let's cache them now
                         // This should probably be done more globally, because large fires will re-do it a lot
                         auto neighs = get_neighbors( p );
-
+                        // get the neighbours that are allowed due to wind direction
+                        double raddir = ( ( winddirection + 180 ) % 360 ) * ( M_PI / 180 );
```

Comment:
Isn't that the same code as at lines 686 (at least the following 20 lines)?
Couldn't it be made into a function?